gcc/testsuite/rust/debug/win64-abi.rs

```diff
@@ -0,0 +1,11 @@
+pub extern "win64" fn square(num: i32) -> i32 {
+    num * num
+}
+
+fn main() {
```

Comment:
Nice check for the specific target i was worried this test will fail on other arch's.